lib/icasework/resource.rb

```diff
@@ -97,7 +98,7 @@ def response
 
     def parser
       lambda do |response, _request, _result|
-        parse_format(response)
+        Data.process(parse_format(response))
```

Comment:
I don't think its a huge issue, but it would be more object-oriented to make this an instance with methods rather than a function bag:

```ruby
Data.new(parse_format(response)).symbolize_keys
```